provision/testdata/flavor_cko_calico-3.23.2.inp.yaml

```diff
@@ -1,4 +1,5 @@
 aci_config:
+  system_id: calico-cni
```

Comment:
Should we not have a UTs with and without the system_id (to avoid getting into the issue we noticed while testing)?

If we do, perhaps adding the system_id would be an overrides case?

Reply:
I have an UT in place for this, will rename it to override

Reply:
Added base case and override test cases for flavor calico